src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -2590,16 +2629,99 @@ public override object PayloadValue(int index)
             }
         }
 
-        private event Action<GenAwareBeginTraceData> m_target;
+        private event Action<GenAwareTemplateTraceData> m_target;
         #endregion
     }
-    public sealed class GenAwareEndTraceData : TraceEvent
+
+    public sealed class GCLOHCompactValues
     {
-        public int Count { get { return GetInt32At(0); } }
-        public int ClrInstanceID { get { return GetInt16At(4); } }
+        /// <summary>
+        /// TODO
+        /// </summary>
+        public int TimePlan
+        {
+            get
+            {
+                return BitConverter.ToInt32(m_dataArray, 0);
```

Comment:
I'm wondering why you are using `BitConverter` instead of the standard `GetInt16At`, `GetInt32At` like all the other places in this file?